frontend-beeyond/app/src/app/core/side-navigation/side-navigation.component.html

```diff
@@ -33,8 +33,35 @@
       </div>
       <div class="spacer"></div>
       <div>
-        <span class="tiny-title">Notifications: {{ actualAgenda.length }}</span>
-        <div class="box"></div>
+        <span class="tiny-title">Notifications: {{ notifications.length }}</span>
+        <div class="box">
+          <div *ngFor="let notification of notifications; let i = index">
+            <table>
+              <tr [routerLink]="'/profile/review/' + notification.entityId" style="cursor: pointer">
+                <td>
+                  <mat-icon
+                    *ngIf="notification.status == notificationStatus.POSITIVE"
+                    style="color: #00ca4e; box-shadow: 0 0 7px 0 #00ca4e; border-radius: 50%"
+                    >check_circle_outline
+                  </mat-icon>
+                  <mat-icon
+                    *ngIf="notification.status == notificationStatus.NEUTRAL"
+                    style="color: #ffbd44; box-shadow: 0 0 7px 0 #ffbd44; border-radius: 50%"
+                    >remove_circle_outline
```

Comment:
I think we should use an "information i in circle" icon as in https://materialdesignicons.com/icon/information-outline instead of the current one.

Reply:
Update: Guess we'll use the info icon as information-outline isn't listed in the mat-icons